README.md

```diff
@@ -202,22 +202,32 @@ There are a number of URL parameters that can aid in testing:
 |Parameter  |Value(s)                  |Description|
 |-----------|--------------------------|-----------|
 |`appMode`  |`dev`, `qa`, `test`       |Unsecured modes that are partitioned off from authenticated sections of the database.|
-|`unit`     |`sas`, `msa`, etc.        |Abbreviated code for selecting a curriculum unit.|
+|`unit`     |`sas`, `msa`, etc.        |Abbreviated code or URL for the curriculum unit.|
 |`problem`  |`2.1`, `3.2`, etc.        |Reference to individual problem in curriculum unit.|
 |`demo`     |none                      |Launches demo creator UI|
 |`demoName` |string (default: `CLUE`)  |Used to partition the demo portion of the database.|
 |`network`  |string                    |Specify the network with which a teacher user is affiliated.|
 |`fakeClass`|string                    |Class id for demo, qa, or test modes.|
-|`fakeUser` |`(student|teacher):<id>`  |Configure user type and (optionally) id.|
+|`fakeUser` |`(student, teacher):<id>`  |Configure user type and (optionally) id.|
```

Comment:
Out of curiosity, do you know if something like `\|` would work here?

Reply:
Yes that does seem to work. Apparently the table escaping has precedence over the backtick escaping. 🤷